tests/p2p_sync_throttle_test.py

```diff
@@ -144,7 +144,7 @@ def extractPrometheusMetric(connID: str, metric: str, text: str):
                 time.sleep(0.5)
                 continue
             Print('Throttling Node Start State')
-            throttlingNodePortMap = {port: id for id, port in connPorts if id != '' and port != '9877'}
+            throttlingNodePortMap = {port: id for id, port in connPorts if port != '0' and port != '9877'}
```

Comment:
should `and` be `or`?

Reply:
My bad. `and` was correct.

Reply:
No. You `or` would allow `9877` as `9877` would be `!= 0`.